packages/core/src/Utils.ts

```diff
@@ -259,3 +273,118 @@ export declare interface DummyServer {
 }
 
 export class DummyServer extends EventEmitter {}
+
+export async function reloadFromCache() {
```

Comment:
This method is mostly duplicated from ` handleCreateRoom()`. I suggest moving this method to `MatchMaker.ts` and call `handleCreateRoom` instead, possibly passing a third argument to specify the room is being "re-loaded".